src/main/java/org/jboss/xavier/integrations/migrationanalytics/business/VMWorkloadInventoryCalculator.java

```diff
@@ -131,11 +134,40 @@ private Long getDiskSpaceList(Map vmStructMap) {
             return ((Number) used_disk_storage).longValue();
         }
 
-
         List<Number> hardwareDisksList = readListValuesFromExpandedEnvVarPath(DISKSIZEPATH, vmStructMap);
         return hardwareDisksList.stream().filter(Objects::nonNull).mapToLong(Number::longValue).sum();
     }
 
+
+    private Boolean getCpuHotAdd(Map vmStructMap) {
+        // If the VM.cpu_hot_add_enabled is present use it, if not set value to null
+
+        Object cpuHotAddPathObject = getValueForExpandedPathAndHandlePathNotPresent(CPUHOTADDENABLEDPATH, vmStructMap,
+                "Setting value to null.");
+
+        return cpuHotAddPathObject != null ? (Boolean) cpuHotAddPathObject : null;
+
+    }
+
+    private Boolean getCpuHotRemove(Map vmStructMap) {
+        // If the VM.cpu_hot_remove_enabled is present use it, if not set value to null
+
+        Object cpuHotRemovePathObject = getValueForExpandedPathAndHandlePathNotPresent(CPUHOTREMOVEENABLEDPATH, vmStructMap,
+                "Setting value to null.");
+
+        return cpuHotRemovePathObject != null ? (Boolean) cpuHotRemovePathObject : null;
+
+    }
+
+    private Boolean getMemoryHotAdd(Map vmStructMap) {
+        // If the VM.memory_hot_add_enabled is present use it, if not set value to null
+
+        Object memoryHotAddPathObject = getValueForExpandedPathAndHandlePathNotPresent(MEMORYHOTADDENABLEDPATH, vmStructMap,
+                "Setting value to null.");
+
+        return memoryHotAddPathObject != null ? (Boolean) memoryHotAddPathObject : null;
+    }
+  
```

Comment:
@m-brophy The content of `getCpuHotAdd, getCpuHotRemove, getMemoryHotAdd, and getCpuAffinity` are exactly the same, do you think it is worth having a single method and pass the PATH variable as a parameter?

Reply:
Done